src/components/UserProfile/UserProfileActions.js

```diff
@@ -57,6 +57,8 @@ export const deleteUser = () => executeQuery( {
 export const userSignedIn = cognitoUser => (dispatch, getState) =>{
     dispatch({ type: USER_SIGNED_IN, cognitoUser })
     dispatch(fetchUserProfile())
+
+    return Promise.resolve();
```

Comment:
What is the purpose of this?

Reply:
It's just for making this function async. It worked but maybe it won't necessary according to rethinking approach.